components/application-broker/internal/storage/populator/instances.go

```diff
@@ -69,7 +69,11 @@ func (p *Instances) Do(ctx context.Context) error {
 	for _, si := range serviceInstances {
 		if si.Spec.ServiceClassRef != nil {
 			if _, ex := abClassNames[si.Spec.ServiceClassRef.Name]; ex {
-				if err := p.inserter.Insert(p.converter.MapServiceInstance(si)); err != nil {
+				convertedInstance, err := p.converter.MapServiceInstance(si)
+				if err != nil {
+					return errors.Wrap(err, "while converting SC service instance to internal service instance model")
```

Comment:
The message is misleading. It should be just something like: "while converting service instance to internal service". There is nothing about service class